src/files-regular/get.js

```diff
@@ -1,52 +1,48 @@
 'use strict'
 
-const promisify = require('promisify-es6')
+const configure = require('../lib/configure')
+const tarStreamToObjects = require('../utils/tar-stream-to-objects')
+const IsIpfs = require('is-ipfs')
```

Comment:
```suggestion
const isIpfs = require('is-ipfs')
```

Reserve Capital for Constructors

Reply:
Maybe it's my Java showing but this looks natural to me 🤣 

Isn't this the sort of thing we have a linter for?

Reply:
wanna add that rule?